### DISCUSSION TITLE
Welcome new committer: hzh0425

### DISCUSSION THREAD

**Discussion by fengjiachun** [sofastack/sofa-jraft#828] [Announcements]
Hi Community,

On behalf of the SOFAJRaft PMC, I am pleased to announce that [hzh0425](https://github.com/hzh0425)  , has been voted in as SOFAJRaft committer.

Please join me to say congratulations to him!

![image](https://user-images.githubusercontent.com/3860496/167606075-a1507214-5630-4ef1-b40e-26040ce6ccbd.png)

Best Regards

**Comment by khotyn** [comment]
Congrats!
